### ISSUE TITLE
[hdpowerview] HD PowerView binding starts in an uninitialized state due to a classNotFoundException

### ISSUE THREAD

**Issue by rogierhofboer** [openhab/openhab-addons#8881]
The HD PowerView binding starts in an uninitialized state on openHAB 3.0.0-M1 due to a classNotFoundException.
It goes online after disabling and enabling the HD PowerView bridge.

## Expected Behavior
HD PowerView bridge should be online after startup

## Current Behavior
HD PowerView bridge starts uninitialized on startup, but does go online after disable and enable.
During startup an exception is thrown:

```
2020-10-27 18:50:30.677 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while calling thing handler factory 'org.openhab.binding.hdpowerview.internal.HDPowerViewHandlerFactory@1acf6c52': java.lang.ClassNotFoundException
java.lang.RuntimeException: java.lang.ClassNotFoundException
	at javax.ws.rs.client.ClientBuilder.newBuilder(ClientBuilder.java:50) ~[?:?]
	at javax.ws.rs.client.ClientBuilder.newClient(ClientBuilder.java:55) ~[?:?]
	at org.openhab.binding.hdpowerview.internal.handler.HDPowerViewHubHandler.<init>(HDPowerViewHubHandler.java:71) ~[?:?]
	at org.openhab.binding.hdpowerview.internal.HDPowerViewHandlerFactory.createHandler(HDPowerViewHandlerFactory.java:51) ~[?:?]
	at org.openhab.core.thing.binding.BaseThingHandlerFactory.registerHandler(BaseThingHandlerFactory.java:129) ~[bundleFile:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.doRegisterHandler(ThingManagerImpl.java:632) [bundleFile:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.registerHandler(ThingManagerImpl.java:609) [bundleFile:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.registerAndInitializeHandler(ThingManagerImpl.java:1078) [bundleFile:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.lambda$10(ThingManagerImpl.java:1060) [bundleFile:?]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) [?:?]
	at java.util.concurrent.CopyOnWriteArraySet.forEach(CopyOnWriteArraySet.java:425) [?:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.lambda$9(ThingManagerImpl.java:1057) [bundleFile:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) [?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177) [?:?]
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) [?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) [?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) [?:?]
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) [?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) [?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) [?:?]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497) [?:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.handleThingHandlerFactoryAddition(ThingManagerImpl.java:1056) [bundleFile:?]
	at org.openhab.core.thing.internal.ThingManagerImpl.onReadyMarkerAdded(ThingManagerImpl.java:1044) [bundleFile:?]
	at org.openhab.core.internal.service.ReadyServiceImpl.lambda$0(ReadyServiceImpl.java:52) [bundleFile:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) [?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) [?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177) [?:?]
	at java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1746) [?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) [?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) [?:?]
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) [?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) [?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) [?:?]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497) [?:?]
	at org.openhab.core.internal.service.ReadyServiceImpl.notifyTrackers(ReadyServiceImpl.java:79) [bundleFile:?]
	at org.openhab.core.internal.service.ReadyServiceImpl.markReady(ReadyServiceImpl.java:52) [bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker.registerReadyMarker(XmlDocumentBundleTracker.java:419) [bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker.finishBundle(XmlDocumentBundleTracker.java:362) [bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker.processBundle(XmlDocumentBundleTracker.java:385) [bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker$2.run(XmlDocumentBundleTracker.java:347) [bundleFile:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException
	at javax.ws.rs.client.ClientFinder.newInstance(ClientFinder.java:120) ~[?:?]
	at javax.ws.rs.client.ClientFinder.find(ClientFinder.java:96) ~[?:?]
	at javax.ws.rs.client.ClientBuilder.newBuilder(ClientBuilder.java:44) ~[?:?]
	... 45 more
Caused by: java.lang.InstantiationException
	at jdk.internal.reflect.InstantiationExceptionConstructorAccessorImpl.newInstance(InstantiationExceptionConstructorAccessorImpl.java:48) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:490) ~[?:?]
	at java.lang.Class.newInstance(Class.java:584) ~[?:?]
	at javax.ws.rs.client.ClientFinder.newInstance(ClientFinder.java:116) ~[?:?]
	at javax.ws.rs.client.ClientFinder.find(ClientFinder.java:96) ~[?:?]
	at javax.ws.rs.client.ClientBuilder.newBuilder(ClientBuilder.java:44) ~[?:?]
	... 45 more
```

## Steps to Reproduce (for Bugs)
1. Startup openHAB with HD PowerView binding installed 
2. Go to Things and the HD PowerView bridge settings. The bridge is uninitialized
3. Select Disable
4. Select Enable
5. It is online

## Context
Starting up 
## Your Environment

* Version used: openHAB 3.0.0-M1
* Environment name and version: Azul 11 x64 
* Operating System and version: Windows 10

**Comment by mstroeve** [comment]
@andrewfg i’am experiencing the same issue while testing the 3.x branche. Do you have any idea? As you have succesfully enhanced the binding in the 2.5 branche  😁👌🏻

**Comment by andrewfg** [comment]
@mstroeve thank you for the feedback; leave it with me; I will try to fix it

**Comment by andrewfg** [comment]
I think this is the same issue as https://github.com/openhab/openhab-addons/issues/7775 -- the binding makes a call below to the package `javax.ws.rs.client` -- this works in OH 2.5.x but apparently not in OH 3.0.x.

@kaikreuzer / @wborn can you fix it?

```
public static Client newClient() {
    return newBuilder().build();
}
```

**Comment by andrewfg** [comment]
#9002 should fix this one.

**Comment by andrewfg** [comment]
Closed with #9002
